uni/lib/controller/local_storage/app_library_reservation.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:sqflite/sqflite.dart';
+import 'package:uni/model/entities/library_reservation.dart';
+
+import 'package:uni/controller/local_storage/app_database.dart';
+
+class LibraryReservationDatabase extends AppDatabase {
+  LibraryReservationDatabase()
+    : super('reservations.db', 
+    [
+      '''CREATE TABLE RESERVATION(
+        id INTEGER PRIMARY KEY AUTOINCREMENT,
+        room TEXT,
+        startDate INT,
```

Comment:
Make the startDate use the [ISO 8601](https://api.flutter.dev/flutter/dart-core/DateTime/toIso8601String.html) standard instead of an unix epoch to improve database consistency between models, since Exams already use it and Schedules will use it.